src/ai/backend/manager/models/user.py

```diff
@@ -657,10 +657,18 @@ async def _post_func(conn: SAConnection, result: Result) -> Row:
             model_store_query = sa.select([groups.c.id]).where(
                 groups.c.type == ProjectType.MODEL_STORE
             )
+            default_query = sa.select([groups.c.id]).where(groups.c.name == "default")
+
```

Comment:
It is not certain that the system has the "default" group named `default`.
We create a group named `default` for a development env for easy testing but not all systems have it